custom_components/myskoda/sensor.py

```diff
@@ -76,6 +78,53 @@ def _status(self) -> ChargingStatus | None:
                 return status
 
 
+class Operation(MySkodaSensor):
+    """Report the most recent operation."""
+
+    entity_description = SensorEntityDescription(
+        key="operation",
+        translation_key="operation",
+        device_class=SensorDeviceClass.ENUM,
```

Comment:
Maybe this should be a diagnostic sensor, and maybe even hidden by default?

Reply:
Diagnostic definitely - adding that now. Not sure about hidden. This could be useful for many users and it's harder to discover it if it's hidden by default :)